apps/meeting/src/components/MeetingControls/VideoInputTransformControl.tsx

```diff
@@ -0,0 +1,238 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  Device,
+  isVideoTransformDevice,
+  VideoTransformDevice,
+} from 'amazon-chime-sdk-js';
+import React, { ReactNode, useEffect, useState } from 'react';
+
+import isEqual from 'lodash.isequal';
+
+import { useBackgroundBlur } from 'amazon-chime-sdk-component-library-react';
+import { useBackgroundReplacement } from 'amazon-chime-sdk-component-library-react';
+import { useVideoInputs } from 'amazon-chime-sdk-component-library-react';
+import { useLocalVideo } from 'amazon-chime-sdk-component-library-react';
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import {
+  isOptionActive,
+  videoInputSelectionToDevice,
+} from '../../utils/device-utils';
+import { ControlBarButton } from 'amazon-chime-sdk-component-library-react';
+import { Camera, Spinner } from 'amazon-chime-sdk-component-library-react';
+import { PopOverItem } from 'amazon-chime-sdk-component-library-react';
+import { PopOverSeparator } from 'amazon-chime-sdk-component-library-react';
+import { DeviceType } from '../../types';
+import useMemoCompare from '../../utils/use-memo-compare';
+import { VIDEO_TRANSFORM_OPTIONS } from '../../types/index';
+
+interface Props {
+  /** The label that will be shown for video input control, it defaults to `Video`. */
+  label?: string;
+  /** The label that will be shown for the background blur button, it defaults to 'Enable Background Blur'. */
+  backgroundBlurLabel?: string;
+  /** The label that will be shown for the background replacement button, it defaults to 'Enable Background Replacement'. */
+  backgroundReplacementLabel?: string;
+}
+
+const VideoInputTransformControl: React.FC<Props> = ({
+  label = 'Video',
+  backgroundBlurLabel = 'Enable Background Blur',
+  backgroundReplacementLabel = 'Enable Background Replacement',
+}) => {
+  const meetingManager = useMeetingManager();
+  const { devices, selectedDevice } = useVideoInputs();
+  const { isVideoEnabled, toggleVideo } = useLocalVideo();
+  const { isBackgroundBlurSupported, createBackgroundBlurDevice } = useBackgroundBlur();
+  const { isBackgroundReplacementSupported, createBackgroundReplacementDevice } = useBackgroundReplacement();
+  const [isLoading, setIsLoading] = useState(false);
+  const [dropdownWithVideoTransformOptions, setDropdownWithVideoTransformOptions] = useState<ReactNode[] | null>(null);
+  const [activeVideoDevice, setDevice] = useState<Device | VideoTransformDevice | null>(meetingManager.selectedVideoInputTransformDevice as Device);
```

Comment:
nit: A state setter can be more specific to indicate a state name.
- `setDevice` => `setActiveVideoDevice`

Or you can remove `active` since it's scoped in this component.
- `videoDevice` / `setVideoDevice`

Reply:
done